e2e/rbd.go

```diff
@@ -517,7 +546,104 @@ var _ = Describe("RBD", func() {
 				}
 				// pvc clone is only supported from v1.16+
 				if v.Major > "1" || (v.Major == "1" && v.Minor >= "16") {
+<<<<<<< HEAD
```

Comment:
Some merge conflicts were not resolved.

Reply:
oh, Thanks for pointing will soon update the PR.